GW2EIEvtcParser/EncounterLogic/Raids/W5/SoullessHorror.cs

```diff
@@ -144,12 +160,20 @@ internal override List<PhaseData> GetPhases(ParsedEvtcLog log, bool requirePhase
                 start = c.EndTime;
                 phases.Add(phase);
             }
-            if (fightEnd - start > ParserHelper.PhaseTimeLimit)
+            if (fightEnd - start > PhaseTimeLimit)
             {
                 var lastPhase = new PhaseData(start, fightEnd, "Final");
                 lastPhase.AddTarget(mainTarget);
                 phases.Add(lastPhase);
             }
+
+            // Add Tormented Deads as secondary target to the phases
+            foreach (PhaseData phase in phases)
+            {
+                var tormentedDeads = Targets.Where(x => x.IsSpecies(TrashID.TormentedDead) && Math.Min(phase.End, x.LastAware) - Math.Max(phase.Start, x.FirstAware) > 0 && phase.CanBeSubPhase).ToList();
```

Comment:
phase.IntersectsWindow(x.FirstAware, x.LastAware)

Reply:
oh that's handy, I'll update arkk and dhuum too